pytorch_blade/src/compiler/mlir/converters/mhlo_conversion.cpp

```diff
@@ -350,10 +353,10 @@ ConvertTorchToMhlo(std::shared_ptr<torch::jit::Graph> graph) {
   mlir_module.push_back(unwrap(op));
 
   ::mlir::torch::Torch::TorchLoweringPipelineOptions options;
-  ::mlir::PassManager pm2(
+  ::mlir::PassManager pm(
       &mlir_context, ::mlir::OpPassManager::Nesting::Implicit);
   if (enable_printing) {
-    pm2.enableIRPrinting(
+    pm.enableIRPrinting(
         /*shouldPrintBeforePass*/ [](mlir::Pass*,
                                      mlir::Operation*) { return true; },
         /*shouldPrintAfterPasss*/
```

Comment:
nit: maybe add a flag (e.g. VLOG_IS_ON) to control if we really need to print the IR, in order to reduce the compilation time  in the normal cases.

Reply:
From the current codebase, we use `TORCH_BLADE_MHLO_DEBUG_LOG` env to enable printing, and I agree with you, better to use `GRAPH_DUMP_ENABLED` , which is conform to PyTorch user.